### PR TITLE
New SPARK course, format reword, day 2

### PR DESCRIPTION
**Pull request by leogermond**
Continuation of the work done in #298, itself a follow up of #295
This improves formating, and add dfn for chapters 6-10 of the new SPARK course

### CODE REVIEW

**Review by yannickmoy — DISMISSED** [review]
looks like the same changes as in the other patch, I'm confused...

**Comment by leogermond** [comment]
> looks like the same changes as in the other patch, I'm confused...

Because it's not done and pushed yet ;)

File: courses/spark_for_ada_programmers/7_specification_language.rst [hunk 1]
```diff
@@ -70,26 +70,26 @@ Simple Expressions
...
-* Counterparts of conditional statements
+* Counterparts of **conditional** statements
 
   - *if expressions* are the counterpart of *if statements*
   - *case expressions* are the counterpart of *case statements*
 
-* Expressions over a collection (range or array or...)
+* Expressions over a **collection** (range or array or...)
 
-  - *universally quantified expression* for properties over all elements
```
Comment (yannickmoy) [inline courses/spark_for_ada_programmers/7_specification_language.rst:80]: I thought this needed to be aligned with "Expressions" above to work as sub-bullet points. no?

**Review by yannickmoy — APPROVED** [review]

File: courses/spark_for_ada_programmers/7_specification_language.rst (hunk 1 above)
Comment (leogermond) [reply courses/spark_for_ada_programmers/7_specification_language.rst:80]: it does not but that's an oversight, good catch !

File: courses/spark_for_ada_programmers/7_specification_language.rst [hunk 2]
```diff
@@ -70,26 +70,26 @@ Simple Expressions
...
   - *delta aggregates* express the value of an updated composite object
   - *iterated component associations* express array aggregates where the
-    expression depends on the index
+    expression depends on the **index**
 
 * Structuring expressions
 
-  - *declare expressions* introduce names for local constants
-  - *expression functions* introduce names for common expressions
+ - *declare expressions* introduce **names** for local constants
```
Comment (yannickmoy) [inline courses/spark_for_ada_programmers/7_specification_language.rst:91]: bad alignment

File: courses/spark_for_ada_programmers/7_specification_language.rst [hunk 3]
```diff
@@ -251,24 +250,24 @@ General Iteration Mechanism
 
 * :dfn:`Iteration over positions` uses :ada:`for .. in` syntax
 
-  - Uses cursor type with :ada:`First`, :ada:`Next` and :ada:`Has_Element`
-  - Function :ada:`Element` is not required
+ - Uses cursor type with :ada:`First`, :ada:`Next` and :ada:`Has_Element`
```
Comment (yannickmoy) [inline courses/spark_for_ada_programmers/7_specification_language.rst:253]: bad alignment

**Review by yannickmoy — APPROVED** [review]

**Pull request merged**